lib/main.js

```diff
@@ -10,6 +10,16 @@ const LineEndingRegExp = /\r\n|\n/g
 
 let disposables = null
 
+export let config = {
+  defaultNewFileLineEnding: {
```

Comment:
I think `defaultNewFileLineEnding` is a bit verbose as a name. How do you feel about changing it to `defaultLineEnding`.